core/templates/pages/exploration-editor-page/modal-templates/save-version-mismatch-modal.controller.ts

```diff
@@ -16,6 +16,10 @@
  * @fileoverview Controller for version mismatch modal.
  */
 
+require(
```

Comment:
Ditto

Reply:
No, it wasn't. I've added it because the modal was breaking.